examples/wallet/app/utils/storage.js

```diff
@@ -1,18 +1,43 @@
 // @flow
 import type { AccountKeys } from '../reducers/types';
-import type { Address, Identifier, PrivateKey } from '../models';
+import { aesEncrypt, aesDecrypt } from './decrypt';
 
-export function saveAccountInfoInLocalStorage(keys: AccountKeys): void {
-  Object.keys(keys).forEach(key => localStorage.setItem(key, keys[key]));
+const WALLET_ENCRYPTED_KEYS = 'wallet.encrypted.account.info';
+
+export function isUnlockWalletPasswordCreated(): boolean {
+  const encryptedAccountInfo = localStorage.getItem(WALLET_ENCRYPTED_KEYS);
+  if (encryptedAccountInfo) return true;
+  return false;
+}
+
+export function saveEncryptedAccountInfo(
+  unlockWalletPassword: string,
+  keys: AccountKeys
+): void {
+  const plainTextAccountInfo = JSON.stringify(keys);
+  if (!unlockWalletPassword || unlockWalletPassword === '') {
```

Comment:
the comparison against the emtpy string is redundant, since empty strings are falsy.